.gitignore

```diff
@@ -1,104 +0,0 @@
-# Byte-compiled / optimized / DLL files
```

Comment:
any reason for removing this file?

Reply:
ping

Reply:
no, es cosa de principiante...rs
Sorry!